DigitalLearningSolutions.Data/Models/SectionContent/SectionContent.cs

```diff
@@ -1,30 +1,56 @@
 ﻿namespace DigitalLearningSolutions.Data.Models.SectionContent
 {
+    using System.Collections.Generic;
+
     public class SectionContent
     {
         public string CourseTitle { get; }
         public string SectionName { get; }
-        public int? TimeMins { get; }
+        public int SectionTime { get; set; }
         public int AverageSectionTime { get; }
         public bool HasLearning { get; }
-        public double PercentComplete { get; }
+        public int DiagnosticAttempts { get; set; }
+        public int SectionScore { get; set; }
+        public int MaxSectionScore { get; set; }
+        public string DiagnosticAssessmentPath { get; }
+        public string PostLearningAssessmentPath { get; }
```

Comment:
I'm not sure, I thought all tutorials had paths and so left all the data in the class. I was then planning on having all the logic for whether or not they should be used/displayed on the page will be done in the view model

Reply:
Just ran a query to check, this is nullable so will add that here :D

Reply:
If you look in the db you'll see that both fields can be null. We should make sure to check for that when writing queries so that we can correctly mark things as optional when necessary 🙂